### PR TITLE
Changes to images and about page

### PR DESCRIPTION
**Pull request by akochari**
## Description
This PR resolves [SS-840](https://scilifelab.atlassian.net/browse/SS-840) - changes in the logos and about page. In addition, I removed html templates and fonts that we no longer use in order to avoid any confusion in the future.

## Checklist

_If you're unsure about any of the items below, don't hesitate to ask. We're here to help!
This is simply a reminder of what we are going to look for before merging your code._

- [x] This pull request is against **develop** branch (not applicable for hotfixes)
- [x] I have included a link to the issue on GitHub or JIRA (if any)
- [ ] I have included migration files (if there are changes to the model classes)
- [x] I have included, reviewed and executed tests (unit and end2end) to complement my changes
- [ ] I have updated the related documentation (if necessary)
- [ ] I have updated the release notes (releasenotes.md)
- [x] I have added a reviewer for this pull request
- [x] I have added myself as an author for this pull request
- [ ] In the case I have modified settings.py, then I have also updated the studio-settings-configmap.yaml file in serve-charts

## Further comments

Anything else you think we should know before merging your code!

### CODE REVIEW

File: templates/portal/about.html [hunk 1]
```diff
@@ -10,7 +10,8 @@
...
+            <p>SciLifeLab Serve is developed and maintained by the <a href="https://scilifelab.se/data">SciLifeLab Data Centre</a>. Initially the service was based on <a href="https://github.com/scaleoutsystems/stackn">STACKn</a>, a platform developed by <a href="https://github.com/scaleoutsystems">Scaleout systems</a> (a spinoff from the Department of Information Technology at Uppsala University). Since then, the SciLifeLab Data Centre team has expanded and improved the platform. The most up to date source code behind SciLifeLab Serve is <a href="https://github.com/ScilifelabDataCentre/stackn/">available with an open source license on GitHub</a>. We welcome any other organisations running a similar service using our code.</p>
```
Comment (churnikov) [inline templates/portal/about.html:13]: I think for many links similar or identical to this one we could try to do something like this:

image
image
image

This is inspired by how substitutions work in restructured text
https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#substitutions

File: studio/settings.py [hunk 2]
```diff
@@ -297,7 +297,7 @@
 AUTH_PROTOCOL = "http"
 STUDIO_URL = "http://studio.127.0.0.1.nip.io:8080"
 # To enable sticky sessions for k8s ingress
-SESSION_COOKIE_DOMAIN = ".127.0.0.1.nip.io"
+# SESSION_COOKIE_DOMAIN = ".127.0.0.1.nip.io"
```
[the remark this answers is not in the document]
Comment (akochari) [reply studio/settings.py]: ah no, this was not supposed to be committed, thanks for noticing. uncommented now.

File: static/images/logos/apps/vscode-logo.png
[the remark this answers is not in the document]
Comment (akochari) [reply static/images/logos/apps/vscode-logo.png:1]: that's a good idea! I now replaced all logos that we have with .svg files.

File: templates/portal/about.html (hunk 1 above)
Comment (akochari) [reply templates/portal/about.html:13]: I didn't know about this as an option before so thank you for bringing it up! I'd say links directly in html like this are sufficient for us though, we don't use links excessively.

File: static/images/logos/apps/dashapp-logo.svg
Comment (churnikov) [inline static/images/logos/apps/dashapp-logo.svg:1]: Why does it look like it was sketched?

**Review by churnikov — APPROVED** [review]

**Pull request merged**